_inc/client/lib/analytics/index.js

```diff
@@ -102,22 +108,30 @@ const analytics = {
 
 	tracks: {
 		recordEvent: function( eventName, eventProperties ) {
-			let superProperties;
+			let blogIdProperty,
+				superProperties;
 
 			eventProperties = eventProperties || {};
 
-			debug( 'Record event "%s" called with props %s', eventName, JSON.stringify( eventProperties ) );
 			if ( eventName.indexOf( 'akismet_' ) !== 0 && eventName.indexOf( 'jetpack_' ) !== 0 ) {
 				debug( '- Event name must be prefixed by "akismet_" or "jetpack_"' );
 				return;
 			}
 
+			if ( _blog ) {
+				blogIdProperty = _blog;
+				debug( '- Blog Id: %o', _blog );
+				eventProperties = assign( eventProperties, blogIdProperty );
+			}
+
 			if ( _superProps ) {
 				superProperties = _superProps.getAll();
 				debug( '- Super Props: %o', superProperties );
 				eventProperties = assign( eventProperties, superProperties );
 			}
 
+			debug( 'Record event "%s" called with props %s', eventName, JSON.stringify( eventProperties ) );
```

Comment:
👍 It's great that we're moving this to the bottom, so we'll be able to debug all the props we're actually logging, and not just a limited set of them.